app/src/main/cpp/skyline/kernel/svc.cpp

```diff
@@ -176,6 +176,7 @@ namespace skyline::kernel::svc {
         if (!sourceObject)
             throw exception("svcUnmapMemory: Cannot find source memory object in handle table for address 0x{:X}", source);
 
+        state.process->memory.FreeMemory(sourceChunk.value().memory->guest);
```

Comment:
Don't go through chunk here, literally use the svc supplied dest/size, same for unMapPhysicalMemory